src/styles/components/tile.scss

```diff
@@ -31,12 +35,32 @@ $tile: '.tile';
     margin: 0;
```

Comment:
Not sure what's caused this change, but the spacing between header and subheader is too narrow now. Eyeballing it at `$global-margin/2`

Reply:
I think it now has correct spacing